config

```diff
@@ -1,4 +1,4 @@
 baseuri     https://nodejs.org/dist
-default_variant stretch
-alpine_version  3.11
+default_variant buster
```

Comment:
Ah, looking at the stackbrew diff I now see what you meant here @nschonni. I wanted to change the default for v15, not the other versions. Should we change it back but overwrite for v15?

Reply:
Ah, yeah that would make sense

Reply:
I don't know bash enough to be able to fix `get_config` to allow being overwritten by a local file

Reply:
@nodejs/docker anyone with bash-fu up for fixing it? If not I recommend rolling back the config changes so we can land it. 15.0.1 is out as well, fwiw